x-pack/plugins/security_solution/server/lib/detection_engine/routes/privileges/read_privileges_route.test.ts

```diff
@@ -58,38 +62,22 @@ describe('read_privileges route', () => {
       clients.clusterClient.callAsCurrentUser.mockImplementation(() => {
         throw new Error('Test error');
       });
-      const response = await server.inject(getPrivilegeRequest(), context);
+      const response = await server.inject(
+        getPrivilegeRequest({ auth: { isAuthenticated: false } }),
+        context
+      );
       expect(response.status).toEqual(500);
       expect(response.body).toEqual({ message: 'Test error', status_code: 500 });
     });
 
     it('returns 404 if siem client is unavailable', async () => {
       const { securitySolution, ...contextWithoutSecuritySolution } = context;
-      const response = await server.inject(getPrivilegeRequest(), contextWithoutSecuritySolution);
+      const response = await server.inject(
+        getPrivilegeRequest({ auth: { isAuthenticated: false } }),
+        contextWithoutSecuritySolution
+      );
       expect(response.status).toEqual(404);
       expect(response.body).toEqual({ message: 'Not Found', status_code: 404 });
     });
   });
-
-  describe('when security plugin is disabled', () => {
```

Comment:
__note:__ there is no explicit dependency on the security plugin in this code anymore, implicit dependency is already covered.